apps/server/src/modules/files-storage/client/s3-client.adapter.spec.ts

```diff
@@ -1,11 +1,11 @@
 import { S3Client } from '@aws-sdk/client-s3';
+import { Upload } from '@aws-sdk/lib-storage';
 import { createMock, DeepMocked } from '@golevelup/ts-jest';
-import { InternalServerErrorException, NotFoundException } from '@nestjs/common';
+import { InternalServerErrorException } from '@nestjs/common';
 import { Test, TestingModule } from '@nestjs/testing';
 import { Logger } from '@src/core/logger';
 import { Readable } from 'node:stream';
 import { FileDto } from '../dto';
-import { ICopyFiles } from '../interface';
 import { S3Config } from '../interface/config';
 import { S3ClientAdapter } from './s3-client.adapter';
 
```

Comment:
@bischofmax Du sagtest doch, dass du keine Variablen mehr außerhalb von setup-Funktionen haben willst, oder? Aber hier sind config, pathToFile und bytesRange auf dem top Level definiert. Andererseits wird der path für den create-Test separat erzeugt in createFileAndPath.

Ich bin nicht dagegen. Zum Beispiel die Tests zu moveToTrash werden dadurch ja viel kürzer und übersichtlicher. Ich frage nur aus Verständnisgründen.